frontend/src/app/shared/components/file-links/file-picker-modal/file-picker-modal.component.ts

```diff
@@ -90,17 +106,21 @@ export class FilePickerModalComponent extends OpModalComponent implements OnInit
     this.breadcrumbs = new BreadcrumbsContent([{
       text: this.locals.storageName as string,
       icon: getIconForStorageType(this.locals.storageType as string),
+      navigate: () => this.changeLevel(null, this.breadcrumbs.crumbs.slice(0, 1)),
     }]);
 
-    const filesLink:IHalResourceLink = {
-      href: `${(this.locals.storageLink as IHalResourceLink).href}/files`,
-      title: 'Storage files',
-    };
+    this.storageFiles$
+      .pipe(this.untilDestroyed())
+      .subscribe((files) => {
```

Comment:
Should this be unsubscribed in `ngOnDestroy` as well?

Reply:
this is taken care of by `this.untilDestroyed()`